druid-shell/src/window.rs

```diff
@@ -367,43 +332,34 @@ pub struct WindowBuilder(platform::WindowBuilder);
 impl WindowBuilder {
     /// Create a new `WindowBuilder`.
     ///
-    /// Takes the [`Application`] that this window is for.
-    ///
-    /// [`Application`]: struct.Application.html
+    /// Takes the [`Application`](crate::Application) that this window is for.
     pub fn new(app: Application) -> WindowBuilder {
         WindowBuilder(platform::WindowBuilder::new(app.platform_app))
     }
 
-    /// Set the [`WinHandler`]. This is the object that will receive
-    /// callbacks from this window.
-    ///
-    /// [`WinHandler`]: trait.WinHandler.html
+    /// Set the [`WinHandler`]. This is the object that will receive callbacks from this window.
```

Comment:
Although we aren't always the best at this, a good rule of thumb for docstrings is to treat them like git commits; A very short, maximally descriptive line, with everything else after a newline. The rationale here is that the first line shows up in the rendered docs as the description of an item in a module, for instance.